ui/src/app/components/confirm-delete-modal.tsx

```diff
@@ -42,94 +40,34 @@ export class ConfirmDeleteModal extends React.Component<
     this.props.receiveDelete();
   }
 
-  validateDeleteText = (event) => {
-    event.toLowerCase().match('delete')
-      ? this.setState({ deleteDisabled: false })
-      : this.setState({ deleteDisabled: true });
-  };
```

Comment:
Move this to confirm-workspace-delete-modal

Reply:
Nice refactoring!